addons/a11y/package.json

```diff
@@ -22,6 +22,9 @@
   "license": "MIT",
   "main": "dist/index.js",
   "types": "dist/index.d.ts",
+  "typesVersions": {
+    "<=3.5": { "*": ["ts3.5/*"] }
```

Comment:
Maybe to update, after discussing https://github.com/storybookjs/storybook/pull/9847#discussion_r379344314